### DISCUSSION TITLE
Is there an option to receive email notifications on a new comment?

### DISCUSSION THREAD

**Discussion by ddengster** [posativ/isso#920] [Q&A] [answered]
Hi, I'm currently evaluating isso as a commenting system for my blog, is there an option to receive email notifications on a new comment?

Also, if I reply to a comment, will users be able to receive an email notification if they registered their email?

**Comment by BBaoVanC** [comment, marked as answer]
> is there an option to receive email notifications on a new comment?

Yes, if you set up the SMTP section so Isso can send emails, and then add your own email under the `to` field, then it should send you emails for every new comment on the entire site, with links to the admin page too.

> Also, if I reply to a comment, will users be able to receive an email notification if they registered their email?

There's a checkbox to enable those notifications. By default it's unchecked so the user has to manually check the box, but you can set the `data-isso-reply-notifications-default-enabled` option (see [Client Configuration docs](https://isso-comments.de/docs/reference/client-config/)) to `true` to make that box checked by default.
